lib/cc/cli/command.rb

```diff
@@ -53,6 +57,14 @@ def rainbow
         @rainbow ||= Rainbow.new
       end
 
+      def uid
+        @uid ||= filesystem.uid
+      end
+
+      def gid
+        @gid ||= filesystem.gid
+      end
```

Comment:
`#uid` and `#guid` should be cheap, with `File.stat` being the only (and not really) expensive call. What do you think about `FileSystem#stat` delegating to and memoizing `File.stat`, then just calling these methods when needed?